packages/next/src/server/base-server.ts

```diff
@@ -2166,6 +2170,11 @@ export default abstract class Server<
     const hasDebugFallbackShellQuery =
       hasDebugStaticShellQuery && query.__nextppronly === 'fallback'
 
+    const skipPostponed = shouldSkipPostponedState(
+      req,
+      this.renderOpts.experimental.streamingMetadata
+    )
```

Comment:
probably best to rename these functions and variables to something like `isHTMLBotRequest`. It makes the rest of the logic easier to understand why we have this special case. Saying we should skip postpone doesn't really help understand when we want to skip the postpone